privacy-configuration/tests.json

```
@@ -175,6 +176,19 @@
                 "expectFeatureEnabled": false,
                 "exceptPlatforms": [
                     "safari-extension",
+                    "windows-browser",
+                    "android-browser"
+                ]
+            },
+            {
+                "name": "eme should be disabled",
+                "featureName": "eme",
+                "siteURL": "https://example.org",
+                "expectFeatureEnabled": false,
+                "exceptPlatforms": [
+                    "safari-extension",
+                    "ios-browser",
+                    "macos-browser",
```

Web extension also doesn't have that feature AFAIK:

```suggestion
                    "macos-browser",
                    "web-extension",
```

---

Done!